CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ## Fixes
 - fix calling return type getter function `@Field(type => Foo)` before finishing module evaluation (allow for extending circular classes using `require`)
 - fix nullifying other custom method decorators - call the method on target instance, not the stored reference to original function (#247)
+- skip superfluous type-conversion. (#229)
```

Comment:
- No `.` at the end.
- Let's make it more developer friendly message - "Prevent unnecessary conversion of object that are instance of object type class" or something like that. Write what you would like to read in the changelog after installing new TypeGraphQL version from npm.